### ISSUE TITLE
Increase or decrease line spacing depending on used TTF

### ISSUE THREAD

**Issue by soenkegissel** [hanjoongcho/aaf-easydiary#13]
If another TTF than the predefined is used the line spacing can get to little or to big. It would be nice to have a setting for increasing or decreasing the line spacing. Most likely no need to add this option to the entry itself, since the setting is made only once.
